libs/components/src/Typography.tsx

```diff
@@ -0,0 +1,23 @@
+import { Serif, Sans, SerifProps, SansProps } from "@artsy/palette";
+import React, { FC } from "react";
+
+type SerifPreset = Omit<SerifProps, "size" | "element">;
+type SansPreset = Omit<SansProps, "size" | "element">;
+
+export const H1: FC<SerifPreset> = ({ children, ...props }) => (
+  <Serif size="10" element="h1" my={3} {...props}>
+    {children}
+  </Serif>
+);
+
+export const H2: FC<SansPreset> = ({ children, ...props }) => (
+  <Sans size="8" element="h2" mt={2} {...props}>
+    {children}
+  </Sans>
+);
+
+export const P: FC<SansPreset> = ({ children, ...props }) => (
+  <Sans size="3" element="p" {...props}>
+    {children}
+  </Sans>
+);
```

Comment:
We discussed moving these items to palette recently, might as well do that now?

Reply:
I'm not sure what the actual levels in palette should be. This is mostly just for convenience for now. Like all of these components, when palette supports something I'm happy to remove it from here.

Reply:
The less we have in studio core from the very onset the better. Meaning, unless we _need_ H1 components and they're currently shared across multiple files, we might as well just use `<Sans>` directly, as we do elsewhere in our apps, and then when the items are created in palette (which will be soon) we can refactor here.

Reply:
I'm not going to touch these for now. They won't live here long term, but they're used in several places for simplicity at the moment. Once we port something similar to palette I'll delete these in favor of palette's version.